akka-persistence-tck/src/main/scala/akka/persistence/japi/journal/JavaJournalPerfSpec.scala

```diff
@@ -43,7 +44,8 @@ import org.scalatest.Informer
  */
 class JavaJournalPerfSpec(config: Config) extends JournalPerfSpec(config) {
   override protected def info: Informer = new Informer {
-    override def apply(message: String, payload: Option[Any]): Unit = System.out.println(message)
+    override def apply(message: String, payload: Option[Any])(implicit pos: Position): Unit =
```

Comment:
I guess now you can print the Position if you want. Not sure what this method does here so I don't know what do do with it.